src/vscode-bicep/src/tree/AzManagementGroupTreeItem.ts

```diff
@@ -32,6 +32,13 @@ export class AzManagementGroupTreeItem extends AzExtParentTreeItem {
     );
 
     const managementGroupInfos = managementGroupsAPI.managementGroups.list();
+
+    if (managementGroupInfos.byPage.length == 0) {
+      throw new Error(
+        "You do not have access to any management group. Please create one in azure portal and try deploy again"
```

Comment:
```suggestion
        "You do not have access to any management group. Please create one in the Azure portal and try to deploy again"
```

Reply:
Updated here:
https://github.com/Azure/bicep/commit/a5b5d0f1ebb4485586a47937c2c90327e1c799f4

Thank you!